rom/dev/src/rom_env.rs

```diff
@@ -65,9 +64,6 @@ pub struct RomEnv {
     /// PCR Bank
     pub pcr_bank: PcrBank,
 
-    /// FHT Data Store
-    pub fht_data_store: FhtDataStore,
```

Comment:
This also seems unrelated to upgrading to Rust 1.83

Reply:
It's unused and rust 1.83 complains about it.